styles/_header.scss

```diff
@@ -0,0 +1,10 @@
+.header {
+  display: flex;
+
+  &__logo {
+    width: 505px;
```

Comment:
Consider using relative units or refrain from adding width on the image itself, making it easier to make it responsive later.

Reply:
I can use relative units in reference to nav (e.g. flex-basis), but we don't have nav right now

Reply:
I think it can be changed later when we join everything together.